src/wormhole/_rendezvous.py

```diff
@@ -61,6 +62,54 @@ def buildProtocol(self, addr):
         return proto
 
 
+def mint_hashcash(bits, resource):
+    "1:6:210623:arbitrary string::9WrCxB1SdCBOM3i5:000005"
+    counter = 0
+    timestamp = "210623"
+    import base64
+    while True:
+        attempt = "1:{}:{}:{}::{}:{}".format(
+            bits,
+            timestamp,
+            resource,
+            base64.b64encode(os.urandom(12)),
+            counter,
```

Comment:
It seem like `counter` is also base64 encoded?

Reply:
this does seem to work .. it'll be ascii just because it's only ever `[0..9]` ... but maybe instead it should be binary (and then base64 encoded)...?
(The `hashcash` binary that comes with Debian also does a weird thing by default where it lower-cases everything .... not sure why, but it has `-C` to turn that off).